defaults.go

```diff
@@ -3,6 +3,7 @@ package sprig
 import (
 	"bytes"
 	"encoding/json"
+	"gopkg.in/yaml.v2"
```

Comment:
Out of curiosity, is there some motivation to use v2 instead of v3? 
Last v2 commit is november 2020 while v3 is May 2022

Reply:
None whatsoever, the autoimport of Goland for some reason went with that instead of v3. I'll fix.